### PR TITLE
Remove cancel buttons from modals

### PR DESCRIPTION
**Pull request by jingpei**
Default to using the close 'x' button for canceling

### CODE REVIEW

**Comment by jingpei** [comment]
@eugenet8k Please review and let me know if all looks well :)

File: src/views/progress-dialog-view.coffee [hunk 1]
```diff
@@ -51,23 +51,28 @@ define (require) ->
...
         error:
           title:
             I18n?.t('error.did_not_work') or
               "Hmm. That didn't seem to work. Try again?"
-          buttons: _([
-            {text: I18n?.t('buttons.cancel') or 'Cancel'
-            className: 'btn-link'}]).concat(
+          buttons: [
               # stealing action button's style and click handler to try again
               _.extend (_.clone _.first _.filter @default?.buttons,
```
Comment (eugenet8k) [inline src/views/progress-dialog-view.coffee]: I think we should move these 2 spaces to the left

Comment (eugenet8k) [inline src/views/progress-dialog-view.coffee]: I think we should move these 2 spaces to the left

**Comment by eugenet8k** [comment]
Looks, please merge commits and cherry-pick it into the gringotts master as one commit. Thanks!

**Pull request merged**